.github/actions/install-app-dependencies/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Install app dependencies
+description: Install app dependencies using yarn and enable caching
+author: 'Technology Studio'
+outputs:
+  cache-dir:
+    description: The yarn cache directory
+    value: ${{ steps.yarn-cache-dir.outputs.dir }}
+  cache-primary-key:
```

Comment:
probably we should have the same pattern also for node_modules 

so two keys returned

* dependency-download-cache-key
* node-modules-cache-key